### PR TITLE
New major release: uikit -> au, colour system, spacing to grid, typography to grid, typographic scale

### PR DESCRIPTION
**Pull request by alex-page**
This is the start of a lot of work to come. 

Moving from the uikit namespace to au and moving to our more accessible and diverse colour system.

### CODE REVIEW

File: packages/core/src/sass/_globals.scss [hunk 1]
```diff
@@ -89,24 +89,24 @@ $AU-color-Focus:                  #f04400 !default;
...
+$AU-color-Border:                 #878787;
+$AU-color-Background:             #f5f5f5;
+$AU-color-Background-dark:        #ebebeb;
+$AU-color-Background-darkest:     #d6d6d6;
 
 // Color sets
-$AU-color-hero-Text:              $AU-color-Background !global;
-$AU-color-hero-Background:        $AU-color-Action !global;
-$AU-color-hero-Border:            $AU-color-Action !global;
+$AU-color-hero-Text:              $AU-color-Background;
```
Comment (dominikwilkowski) [inline packages/core/src/sass/_globals.scss]: you're gona need `global`... no @alex-page ?

File: packages/core/src/sass/_globals.scss [hunk 2]
```diff
@@ -562,7 +568,7 @@ $AU-border-radius: 3px !default;
  *
  * @return {integer}             - The contrast ratio
  */
-@function AU-color-contrast( $forground, $background ) {
+@function AU-color-contrast( $forground, $background, $silent: false ) {
```
Comment (dominikwilkowski) [inline packages/core/src/sass/_globals.scss]: uh very good solution! I didn't think of that

File: packages/core/src/sass/_globals.scss [hunk 3]
```diff
@@ -89,24 +89,24 @@ $AU-color-Focus:                  #f04400 !default;
 $AU-color-Background-light:       #ffffff !default;
 
 // Non-user definable pallet colors
-$AU-color-Muted:                  #6a6a6a !global;
-$AU-color-Border:                 #878787 !global;
-$AU-color-Background:             #f5f5f5 !global;
-$AU-color-Background-dark:        #ebebeb !global;
-$AU-color-Background-darkest:     #d6d6d6 !global;
+$AU-color-Muted:                  #6a6a6a;
```
Comment (dominikwilkowski) [inline packages/core/src/sass/_globals.scss]: you're going to need `!global` no @alex-page ?

File: packages/core/src/sass/_globals.scss (hunk 2 above)
Comment (alex-page) [reply packages/core/src/sass/_globals.scss]: Yeah, didn't want the warnings to go away for good 😄

File: packages/core/src/sass/_globals.scss [hunk 4]
```diff
@@ -83,37 +83,73 @@ $AU-maxwidth: 42em !default; //Answer to life, the universe, and everything (kee
...
-$AU-color-Action:           #00698f !default;
-$AU-color-Focus:            #f04400 !default;
-$AU-color-Background-light: #ffffff !default;
+$AU-color-Text:                   #313131 !default;
+$AU-color-Action:                 #00698f !default;
+$AU-color-Focus:                  #f04400 !default;
+$AU-color-Background-light:       #ffffff !default;
+
+// Non-user definable pallet colors
+$AU-color-Muted:                  #6a6a6a;
```
Comment (dominikwilkowski) [inline packages/core/src/sass/_globals.scss]: They do still need `@global` @alex-page and obviously the color-a11y function

Comment (alex-page) [reply packages/core/src/sass/_globals.scss]: I'll add that to trello, my bad!

File: packages/core/src/sass/_globals.scss [hunk 5]
```diff
@@ -585,12 +587,14 @@ $AU-border-radius: 3px !default;
...
-	$ratio: $ratio;
-
 	@if( $ratio < 4.5 and $silent == false ) {
 		@warn "Houston, we have a problem; contrast ratio of #{ $forground } on #{ $background } is bad, just #{ $ratio }";
 	}
 
+	@if $rounded == true {
+		$ratio: round( $ratio * 10 ) / 10;
+	}
+
```
Comment (dominikwilkowski) [inline packages/core/src/sass/_globals.scss]: do an `@else` he to be more explicit

Comment (alex-page) [reply packages/core/src/sass/_globals.scss]: Added else

Comment (alex-page) [reply packages/core/src/sass/_globals.scss]: We just want a way to `round` the ratio for displaying the color contrast boxes without a long integer. I don't think it was related to the `@if` above.

File: packages/core/tests/site/index.html [hunk 6]
```diff
@@ -170,7 +170,7 @@ <h2>function: AU-pow</h2>
 
 	<code class="au-pow1">5<sup>3</sup> = 125:</code><br>
 	<code class="au-pow2">4<sup>2</sup> = 16:</code><br>
-	<code class="au-pow3">4<sup>-2</sup> = 0.0625:</code><br>
+	<!-- <code class="au-pow3">4<sup>-2</sup> = 0.0625:</code><br> -->
```
Comment (dominikwilkowski) [inline packages/core/tests/site/index.html]: was that because we disabled rounding?

Comment (alex-page) [reply packages/core/tests/site/index.html]: Yeah 👍

Comment (alex-page) [reply packages/core/tests/site/index.html]: Oh this line of code was always commented out. It errors as:

```
@if $exponent < 0 {
	@error "I can see dead people; the AU-pow function only takes positive numbers as the second argument.";
}
```

**Pull request merged**